gradle-plugin/kokoro/release.sh

```diff
@@ -4,13 +4,15 @@ set -o errexit
 set -o xtrace
 
 mkdir -p $HOME/.gradle
-readonly HOME_GRADLE_PROPERTY="$HOME/.gradle/gradle.properties"
+export GRADLE_USER_HOME="$HOME/.gradle"
```

Comment:
> GRADLE_USER_HOME
> Specifies the Gradle user home directory (which defaults to $USER_HOME/.gradle if not set).

https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_environment_variables

I assumed the default value should work fine. However, the default value uses a strange variable `USER_HOME`, which is undefined. (This is not `HOME`.)